readme.md

```diff
@@ -67,6 +67,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [IPFS](https://github.com/ipfs/awesome-ipfs)
 - [Fuse](https://github.com/vinkla/awesome-fuse)
 - [Heroku](https://github.com/ianstormtaylor/awesome-heroku)
+- [Raspberry Pi](https://github.com/thibmaek/awesome-raspberrypi)
```

Comment:
Add a succinct description. See other entries with description for what format to use.